app/images/incognito.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="32px" height="32px" viewBox="0 0 32 32"><g stroke-width="2" transform="translate(0, 0)"><polygon points="3 16 29 16 25 11 7 11 3 16" fill="none" stroke="#444444" stroke-miterlimit="10" stroke-width="2" data-cap="butt" stroke-linecap="butt" stroke-linejoin="miter"></polygon><polyline points="7 11 9 2 23 2 25 11" fill="none" stroke="#444444" stroke-miterlimit="10" stroke-width="2" data-cap="butt" stroke-linecap="butt" stroke-linejoin="miter"></polyline><circle cx="24" cy="25" r="5" fill="none" stroke="#444444" stroke-miterlimit="10" stroke-width="2" data-cap="butt" data-color="color-2" stroke-linecap="butt" stroke-linejoin="miter"></circle><circle cx="8" cy="25" r="5" fill="none" stroke="#444444" stroke-miterlimit="10" stroke-width="2" data-cap="butt" data-color="color-2" stroke-linecap="butt" stroke-linejoin="miter"></circle><path d="M19.32,23.287a4.926,4.926,0,0,0-6.64,0" fill="none" stroke="#444444" stroke-miterlimit="10" stroke-width="2" data-cap="butt" data-color="color-2" stroke-linecap="butt" stroke-linejoin="miter"></path></g></svg>
```

Comment:
Did you get this icon from our icons collection?

Reply:
Yes, from nucleoapp